hal/lib/athena/HAL.cpp

```diff
@@ -265,6 +265,17 @@ static void timerRollover(uint64_t currentTime, HAL_NotifierHandle handle) {
   HAL_UpdateNotifierAlarm(handle, currentTime + 0x80000000ULL, &status);
 }
 
+void HAL_BaseInitialize(int32_t* status) {
+  // image 4; Fixes errors caused by multiple processes. Talk to NI about this
+  nFPGA::nRoboRIO_FPGANamespace::g_currentTargetClass =
+      nLoadOut::kTargetClass_RoboRIO;
+
+  global.reset(tGlobal::create(status));
+  watchdog.reset(tSysWatchdog::create(status));
+
+  std::atexit(HALCleanupAtExit);
```

Comment:
It should be. I'd have to check and see if LV has a way to dynamically unload libraries that are loaded. If not it is safe, if it does it could be a race.